StripeFinancialConnections/StripeFinancialConnections/Source/Native/Shared/CloseConfirmationAlertHandler.swift

```diff
@@ -29,7 +29,7 @@ final class CloseConfirmationAlertHandler {
                 if let businessName = businessName {
                     return String(
                         format: STPLocalizedString(
-                            "You haven’t finished linking your bank account to Stripe and %@.",
+                            "You haven’t finished linking your bank account.",
```

Comment:
Are we sure we want to remove the `%@`?

Doc:
- https://docs.google.com/document/d/1qib1HtHNGfNCthjAMAZOwVFpI99QnHmdeWcKHJ55gnM/edit

(if we do remove the merchant name, which I don't see in the doc atm, this code can be cleaned up a little too

Reply:
you are right, brought it back